swift/BUILDING.md

```diff
@@ -20,36 +20,31 @@ resulting build.
 Ice for Swift builds and runs on macOS and is supported on the platforms listed
 on the [supported platforms] page.
 
-### Slice to Swift Compiler
-
-You need the Slice to Swift compiler to build Ice for Swift and also to use Ice
-for Swift. The Slice to Swift compiler (`slice2swift`) is a command-line tool
-written in C++. You can build the Slice to Swift compiler from source, or
-alternatively you can install an Ice [binary distribution] that includes this
-compiler.
-
 ### Swift Version
 
 Ice for Swift requires Swift 5 or later.
 
 ## Building Ice for Swift
 
-Then open `ice.xcodeproj` with Xcode and build the `Ice macOS` or `Ice iOS`
-targets.
+Building Ice for Swift requires having built Ice for C++ first from the `cpp` directory. This builds the required
+`XCFramework`s that are used by the Swift package.
 
-The test programs for macOS and iOS can be built using `TestDriver macOS` and
-`TestDriver iOS` respectively.
+To build Ice for Swift and its corresponding tests, run the following command from the `swift` directory:
 
-Building the `TestDriver iOS` application to deploy to an iOS device requires
-signing the application with a developer certificate. You need to update the Xcode projects
-to use your Apple development certificates by setting `DEVELOPMENT_TEAM` environment
-variable to the Id of your development team and regenerate the projects with `rake`.
+```shell
+make
+```
+
+The `PLATFORMS` argument can be used to include the [iOS test controller]:
 
 ```shell
-export DEVELOPMENT_TEAM=U4TBVKNQ7F
-rake
+make PLATFORM=all
 ```
 
+### Building with Xcode
+
+`Package.swift` (Ice for Swift source code), `test/Package.swift` (Ice for Swift tests), and `test/ios/TestDriverApp.xcodeproj` (iOS test controller) can be opened and built using Xcode.
```

Comment:
Should it be links to the files?

For example, Package.swift is ice/Package.swift, not ice/swift/Package.swift.